ports/fastrtps/portfile.cmake

```
@@ -1,10 +1,8 @@
-include(vcpkg_common_functions)
-
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO eProsima/Fast-RTPS

```

```suggestion
    REPO eProsima/Fast-DDS

```

The reason the SHA512 changed is that the repo got renamed.